docs/admin/scale.md

```diff
@@ -17,43 +17,79 @@ Coder runs workspace operations in a queue. The number of concurrent builds will
 
-To support 120 concurrent workspace builds, for example:
+We recommend:
+
+- Running `coderd` on a dedicated set of nodes. This will prevent other workloads from interfering with workspace builds. You can use [node selectors](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector), or [taints and tolerations](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/) to achieve this.
+- Disabling autoscaling for `coderd` nodes. Autoscaling can cause interruptions for users, see [Autoscaling](#autoscaling) for more details.
+- (Enterprise-only) Running external provisioners instead of Coder's built-in provisioners (`CODER_PROVISIONER_DAEMONS=0`) will separate the load caused by workspace provisioning on the `coderd` nodes. For more details, see [External provisioners](../admin/provisioners.md#running-external-provisioners).
+
+For example, to support 120 concurrent workspace builds:
 
 - Create a cluster/nodepool with 4 nodes, 8-core each (AWS: `t3.2xlarge` GCP: `e2-highcpu-8`)
 - Run coderd with 4 replicas, 30 provisioner daemons each. (`CODER_PROVISIONER_DAEMONS=30`)
 - Ensure Coder's [PostgreSQL server](./configure.md#postgresql-database) can use up to 2 cores and 4 GB RAM
 
 ## Recent scale tests
 
-| Environment        | Users | Concurrent builds | Concurrent connections (Terminal/SSH) | Coder Version | Last tested  |
-| ------------------ | ----- | ----------------- | ------------------------------------- | ------------- | ------------ |
-| Kubernetes (GKE)   | 1200  | 120               | 10,000                                | `v0.14.2`     | Jan 10, 2022 |
-| Docker (Single VM) | 500   | 50                | 10,000                                | `v0.13.4`     | Dec 20, 2022 |
+| Environment        | Coder CPU | Coder RAM | Database         | Users | Concurrent builds | Concurrent connections (Terminal/SSH) | Coder Version | Last tested  |
+| ------------------ | --------- | --------- | ---------------- | ----- | ----------------- | ------------------------------------- | ------------- | ------------ |
+| Kubernetes (GKE)   | 16 cores  | 64 GB     | db-custom-1-3840 | 1200  | 120               | 10,000                                | `v0.14.2`     | Jan 10, 2022 |
+| Docker (Single VM) | 16 cores  | 64 GB     | Embedded         | 500   | 50                | 10,000                                | `v0.13.4`     | Dec 20, 2022 |
+| Kubernetes (GKE)   | 3 cores   | 12 GB     | db-f1-micro      | 200   | 3                 | 200 (40KB/s web terminal each)        | `v0.24.1`     | Jun 26, 2023 |
+| Kubernetes (GKE)   | 4 cores   | 8 GB      | db-custom-1-3840 | 1500  | 20                | 1,500                                 | `v0.24.1`     | Jun 27, 2023 |
+| Kubernetes (GKE)   | 2 cores   | 4 GB      | db-custom-1-3840 | 500   | 20                | 500 (640KB/s SSH each)                | `v0.27.2`     | Jul 27, 2023 |
```

Comment:
Nice! Let's remove the first 2 scale tests

Reply:
It might also be wroth adding that the scale tests are for reference purposes only and are not meant to be a recommendation on resource sizing.